.github/labels.md

```diff
@@ -0,0 +1,234 @@
+network infrastructure.
+
+#### Label `component:scripting`
+
+The Pull Request requires knowledge about the FOCS language and probably game
+mechanics.  Probably most used in conjunction with `component:game mechanic` and
+`component:art music text assets`.
+
+#### Label `component:translation`
+
+The Pull Request requires knowledge about the game style, game terminology,
+stringtable syntax and the translation process in general. It maybe helpful to
+know the language of the translation to proofread.
+
+#### Label `component:UI`
+
+The Pull request requires knowledge about the game style, the expected user
+experience and graphical APIs used (GiGi and OpenGL).
+
+#### Label `status:cherry-pick for release`
+
+The Pull Request should be applied to the currently open release branch.  Use
+this label sparsely, only for bug fixes, and document a minimal set of commits
+that should be considered by the release manager for cherry-picking.
+Cherry-picking Pull Requests onto release branches should fix bugs, not
+introduce new ones.
+
+#### Label `status:duplicate`
+
+The topic of the Issue is or was already adressed by another Issue.  Add the
+number of the adressing Issue in a comment and close this Issue.  If the
+closed Issue contains new knowledge that is valuable to solve the adressing
+Issue copy it over to the adressing Issue.
+
+#### Label `status:follow up`
+
+The Pull Request discussion showed that the proposed implementation is
+insufficient in some way or the Pull Request in itself is malformed.  Usually
+not required as we prefer rebasing within the same Pull request to update
+the implementation.  When applying this Label reference the new Pull Request
+number in the old one.
+
+#### Label `status:help wanted`
+
+??? I don't think we really need that.  At least I didn't see any Issue were it
+caused someone required to help to magically appear.  This can be handled better
+with `@`-mentions if someones or some groups special knowledge is required.
+
+#### Label `status:invalid`
+
+The Issue contains content that fails to adress it intention.  An example
+would be a feature request wasn't discussed before in the forums or for Pull
+Requests those requests that compete to adress the same topic, but were not
+chosed to be merged.  Those Issues won't be merged.
+
+#### Label `status:merged`
+
+All relevant commits of this Pull Request were merged into the master
+development branch.  This can either happen by merging, cherry-picking or
+rebasing and pushing onto master.  Close the Issue if GitHub didn't closed
+it automatically.
+
+#### Label `status:resolved`
+
+The Issue was solved satisfying, either by answering properly or fixing the
+underlying bug.  Close the Issue.
+
+#### Label `status:testing requested`
+
+The Implementation is ready for testing but can't be tested sufficiently by the
+developer and he request testing by other developers.  Use this in case of
```

Comment:
English does not have a gender neutral third person pronoun.  

In the past it was considered acceptable to use "he", because women were excluded from the political economic and social spheres.  Now to a native speaker using "he" either implies that you are speaking of a specific person or intentionally excluding women. 

Typical modern usages are either:
- use "they" as a gender neutral third person pronoun.
- use "she" and "he" in alternating paragraphs/examples, if there is enough text to support this.
- use "he/she" in every case.